app/templates/admin/content/system-images/list.hbs

```diff
@@ -1,12 +1,16 @@
 <div class="ui grid segment">
-  {{#each data as |subTopic|}}
-    <div class="eight wide column">
-      <h4>{{subTopic.name}}</h4>
-      {{widgets/safe-image isAvatar=true class='ui big image' src=(if subTopic.placeholder.originalImageUrl subTopic.placeholder.originalImageUrl
-        '/images/placeholders/avatar.png')}}
-      <div class="ui hidden divider"></div>
-      <button class="ui button primary" {{action 'openModal' subTopic.placeholder}} id="changebutton">{{t 'Change'}}</button>
-    </div>
-  {{/each}}
+  {{#if data}}
+    {{#each data as |subTopic|}}
+      <div class="eight wide column">
+        <h4>{{subTopic.name}}</h4>
+        {{widgets/safe-image isAvatar=true class='ui big image' src=(if subTopic.placeholder.originalImageUrl subTopic.placeholder.originalImageUrl
+          '/images/placeholders/avatar.png')}}
+        <div class="ui hidden divider"></div>
+        <button class="ui button primary" {{action 'openModal' subTopic.placeholder}} id="changebutton">{{t 'Change'}}</button>
+      </div>
+    {{/each}}
```

Comment:
Use each else syntax.
https://guides.emberjs.com/v1.13.0/templates/displaying-a-list-of-items/#toc_empty-lists